mk/mcu/STM32H5.mk

```diff
@@ -0,0 +1,227 @@
+                stm32h5xx_hal_ltdc_ex.c \
+                stm32h5xx_hal_mdios.c \
+                stm32h5xx_hal_mdma.c \
+                stm32h5xx_hal_mmc.c \
+                stm32h5xx_hal_mmc_ex.c \
+                stm32h5xx_hal_msp_template.c \
+                stm32h5xx_hal_nand.c \
+                stm32h5xx_hal_nor.c \
+                stm32h5xx_hal_opamp.c \
+                stm32h5xx_hal_opamp_ex.c \
+                stm32h5xx_hal_ramecc.c \
+                stm32h5xx_hal_rng.c \
+                stm32h5xx_hal_rtc.c \
+                stm32h5xx_hal_sai.c \
+                stm32h5xx_hal_sai_ex.c \
+                stm32h5xx_hal_sd_ex.c \
+                stm32h5xx_hal_sdram.c \
+                stm32h5xx_hal_smartcard.c \
+                stm32h5xx_hal_smartcard_ex.c \
+                stm32h5xx_hal_smbus.c \
+                stm32h5xx_hal_spdifrx.c \
+                stm32h5xx_hal_spi.c \
+                stm32h5xx_hal_sram.c \
+                stm32h5xx_hal_swpmi.c \
+                stm32h5xx_hal_usart.c \
+                stm32h5xx_hal_usart_ex.c \
+                stm32h5xx_hal_wwdg.c \
+                stm32h5xx_ll_adc.c \
+                stm32h5xx_ll_bdma.c \
+                stm32h5xx_ll_comp.c \
+                stm32h5xx_ll_crc.c \
+                stm32h5xx_ll_dac.c \
+                stm32h5xx_ll_delayblock.c \
+                stm32h5xx_ll_dma2d.c \
+                stm32h5xx_ll_exti.c \
+                stm32h5xx_ll_fmc.c \
+                stm32h5xx_ll_gpio.c \
+                stm32h5xx_ll_hrtim.c \
+                stm32h5xx_ll_i2c.c \
+                stm32h5xx_ll_lptim.c \
+                stm32h5xx_ll_lpuart.c \
+                stm32h5xx_ll_mdma.c \
+                stm32h5xx_ll_opamp.c \
+                stm32h5xx_ll_pwr.c \
+                stm32h5xx_ll_rcc.c \
+                stm32h5xx_ll_rng.c \
+                stm32h5xx_ll_rtc.c \
+                stm32h5xx_ll_swpmi.c \
+                stm32h5xx_ll_usart.c \
+                stm32h5xx_ll_utils.c \
+                stm32h5xx_hal_timebase_rtc_alarm_template.c \
+                stm32h5xx_hal_timebase_rtc_wakeup_template.c \
+                stm32h5xx_hal_timebase_tim_template.c
+
+STDPERIPH_SRC   := $(filter-out ${EXCLUDES}, $(STDPERIPH_SRC))
+
+#USB  ##TODO - need to work through the USB drivers, new directory: USBX
+#USBCORE_DIR = $(ROOT)/lib/main/STM32H5/Middlewares/ST/usbx/Common
+#USBCORE_SRC = $(notdir $(wildcard $(USBCORE_DIR)/Src/*.c))
+#EXCLUDES    =
+#USBCORE_SRC := $(filter-out ${EXCLUDES}, $(USBCORE_SRC))
+
+#VPATH := $(VPATH):$(USBCDC_DIR)/Src:$(USBCORE_DIR)/Src:$(USBHID_DIR)/Src:$(USBMSC_DIR)/Src:$(STDPERIPH_DIR)/src
+
+DEVICE_STDPERIPH_SRC := $(STDPERIPH_SRC) \
+                        $(USBCORE_SRC) \
+                        $(USBCDC_SRC) \
+                        $(USBHID_SRC) \
+                        $(USBMSC_SRC)
+
+#CMSIS
+VPATH           := $(VPATH):$(CMSIS_DIR)/Include:$(CMSIS_DIR)/Device/ST/STM32H5xx
```

Comment:
+= ?

Reply:
> += ?

Agree - but there is a bit of an historical convention